templates/set_version_boundary.cdc

```diff
@@ -0,0 +1,40 @@
+import NodeVersionBeacon from 0xNODEVERSIONBEACONADDRESS
```

Comment:
@janezpodhostnik what should this address be?

Reply:
```suggestion
import NodeVersionBeacon from 0xe467b9dd11fa00df
```

mainnet service account